python/private/local_runtime_repo.bzl

```diff
@@ -46,7 +46,7 @@ def _local_runtime_repo_impl(rctx):
     on_failure = rctx.attr.on_failure
 
     platforms_os_name = repo_utils.get_platforms_os_name(rctx)
-    if not platforms_os_name:
+    if platforms_os_name not in ["linux", "osx", "windows"]:
```

Comment:
You can just remove this block; as mentioned in the originating PR, I just had it return None because I knew there were cases it didn't handle, but didn't know what they were. Now that it handles all the cases, we can just assume whatever value is returned is correct.